docker/puppetserver-standalone/docker-entrypoint.d/90-consul.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+if [ "$CONSUL_ENABLED" = "true" ]; then
+  ipaddress="$(ifconfig eth0 | grep 'inet addr' | cut -d ':' -f 2 | cut -d ' ' -f 1)"
+  consul_hostname="${CONSUL_HOSTNAME:-consul}"
+  consul_port="${CONSUL_PORT:-8500}"
+
+  cat <<SERVICEDEF > /puppet-service.json
+{
+  "name": "puppet",
+  "id": "$HOSTNAME",
+  "port": 8140,
+  "address": "$ipaddress",
+  "checks": [
+    {
+      "http": "https://$HOSTNAME:8140/production/status/test",
+      "tls_skip_verify": true,
+      "header": {"Accept": ["pson"]},
```

Comment:
Ha oh wow, forgot about PSON!

Are we able to specify JSON here instead?

Or is this actually PSON?

Reply:
I copied that from the healthcheck but it looks like the calls pass without specifying any extra headers, so will probably just get rid of that in both places